example/pubspec.lock

```diff
@@ -268,10 +268,10 @@ packages:
     dependency: transitive
     description:
       name: intl
-      sha256: d6f56758b7d3014a48af9701c085700aac781a92a87a62b1333b46d8879661cf
+      sha256: "3bc132a9dbce73a7e4a21a17d06e1878839ffbf975568bc875c60537824b0c4d"
```

Comment:
Please undo changes to pubspec.lock file as these are distinct to your local setup